i18n/README.md

```diff
@@ -13,6 +13,14 @@ they will be handled.
 When adding a string, please make the key as descriptive as possible because
 this is used as context for translators.
 
+Make sure your string value only has plain text and simple HTML.
+
+When using a translation string, please be aware that the it comes from an untrusted 
+source like a translator and may contain security risks. If the translation string 
+contains HTML elements, you should pass `markdown: true` or `markdown: :inline` to the 
+`I18n.t` method to have it parse the translation string as markdown instead. (Markdown 
+is a safe alternative to HTML.)
```

Comment:
I'd add something along the lines of "try to always use plain text. If you want to add formatting, use markdown. If you want to do something that markdown can't do, you can add HTML, but if you want your string to support embedded HTML you _must_ render it as markdown. This is because our markdown renderer will remove any potentially-dangerous html"

In particular, I think it's somewhat misleading to say that markdown is a safe alternative to HTML. It's more accurately the case that markdown provides alternatives to a safe _subset_ of HTML, and that our markdown parsers are in addition to that also able to protect against the _unsafe_ subset of HTML.

Reply:
This is much better! I'll add it to the doc.